README.md

```diff
@@ -77,11 +77,16 @@ A note regarding "open a shell" in the following sections: this implies that the
 * On RHEL-based Linux, run e.g. `source /opt/rh/devtoolset-8/enable`.
 * On macOS, there is usually no action required (the current Xcode command line tools are activated by default).
 
+_Note: on Windows, replace `bin` with `Scripts` in the following commands._
+
 #### Build Python Wheel
 
-1. Open a shell in the PyPRT git root.
-1. First time only: run `pipenv install` to get all required Python packages.
-1. Run `pipenv run python setup.py bdist_wheel`. This will build the CMake project and Python packages.
+1. Open a shell in the PyPRT git root. 
+1. First time only: setup a virtual Python environment with build dependencies for PyPRT.
+    1. Create the virtual environment: `python -m venv pyprt-venv`
+    1. Get latest pip: `pyprt-venv/bin/python -m pip install --upgrade pip`
+    1. Install build dependencies for PyPRT: `pyprt-venv/bin/python -m pip install -r requirements`
```

Comment:
Did you mean `pyprt-venv/bin/python -m pip install -r requirements.txt`?

Reply:
fixed